src/main/java/subway/line/entity/Sections.java

```diff
@@ -0,0 +1,64 @@
+package subway.line.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class Sections {
+
+    @OrderBy("id ASC")
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+    private List<Section> sections = new ArrayList<>();;
```

Comment:
일급 컬렉션 적용 👍 

https://github.com/next-step/atdd-subway-map/pull/1108#discussion_r1485533502
>여기서 삭제 API를 호출할 때는 제거된 것으로 보였는데
아래서 조회할 때는 제거가 안된 것으로 보이네요 ㅠㅠ
어느 부분에서 잘못 되었는지 모르겠어요

로그를 보면 delete 쿼리가 수행되지 않는 걸 확인하실 수 있는데요 
delete 쿼리가 수행되려면 orphanRemoval 옵션을 추가하시면 됩니다 
```suggestion
    @OrderBy("id ASC")
    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Section> sections = new ArrayList<>();;
```

Reply:
아하 자식 Entity만 제거될 경우에 delete 쿼리가 해당 조건이 있어야만 나가는 것이었군요